packages/webawesome/src/styles/native.css

```diff
@@ -859,8 +858,7 @@
     /* Exclude inputs that don't accept text */
     [type='button'],
     [type='checkbox'],
-    [type='color'],
-    [type='file'],
+    [type='color']
```

Comment:
Needs a coma. I added it in my next branch but doesn't seem to be reflected here.